cmd/agent/app/processors/thrift_processor_test.go

```diff
@@ -181,12 +189,12 @@ func assertJaegerProcessorCorrectness(t *testing.T, collector *testutils.MockTCo
 	assertProcessorCorrectness(t, metricsFactory, sizeF, nameF, "jaeger")
 }
 
-func assertZipkinProcessorCorrectness(t *testing.T, collector *testutils.MockTCollector, metricsFactory *metricstest.Factory) {
+func assertZipkinProcessorCorrectness(t *testing.T, collector *testutils.GrpcCollector, metricsFactory *metricstest.Factory) {
 	sizeF := func() int {
-		return len(collector.GetZipkinSpans())
+		return len(collector.GetJaegerBatches())
```

Comment:
Why have these changed to Jaeger batches when in a zipkin processor test? Is zipkin format collection via the agent relevant now? If not, then we should probably change the name of the test.

Reply:
Jaeger agent can injest zipkin thrift spans https://www.jaegertracing.io/docs/1.18/deployment/#agent. However the reporter in the agent convert these spans to jaeger batches.

Reply:
ok